js/main.js

```diff
@@ -1,42 +1,139 @@
+];
 
-function getFloatingPointNumber(min, max, floatPoint) {
-  if (min < 0 || max < 0 || floatPoint < 0) {
-    throw new Error('Числа должны быть больше или равны 0');
-  }
-  if (min === max) {
-    return min;
+const PHOTOLINKS = [
+  'duonguyen-8LrGtIxxa4w.jpg',
+  'brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'claire-rendall-b6kAwr1i0Iw.jpg',
+];
+
+const minLat = 35.65000;
+const maxLat = 35.70000;
+const minLng = 139.70000;
+const maxLng = 139.80000;
+
+const QUANTITYADS = 10;
+// function random number from two integers number
+
+const getRandomNumber = (a, b) => {
+  const lower = Math.ceil(Math.min(Math.abs(a), Math.abs(b)));
+  const upper = Math.floor(Math.max(Math.abs(a), Math.abs(b)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+};
+
+// function random number with floating points from two integers numbers
+
+const getFloatingPointNumber = (a, b, digits = 1) => {
+  const lower = Math.min(Math.abs(a), Math.abs(b));
+  const upper = Math.max(Math.abs(a), Math.abs(b));
+  const result = Math.random() * (upper - lower) + lower;
+  return Number(result.toFixed(digits));
+};
+
+// get random avatar number
+
+const getNumberAvatar = () => {
+  const numberPhoto = String(getRandomNumber(1, 10));
+  return (numberPhoto.length === 1) ? `0${numberPhoto}` : numberPhoto;
+};
+
+//GET RANDOM FEATURES
+// shuffle array
+const shuffle = (arr) => {
+  let j;
+  let temp;
+  for(let i = arr.length - 1; i > 0; i--) {
+    j = Math.floor(Math.random() * (i+1));
+    temp = arr[j];
+    arr[j] = arr[i];
+    arr[i] = temp;
   }
-  if (min > max) {
-    const t = max;
-    max = min;
-    min = t;
+  return arr;
+};
+// random features number
+const getNumberFeatures = () => {
+  const quatityFeatures = getRandomNumber(0, FEATURES.length - 1);
+  return quatityFeatures;
+};
+// get array random features
+const getRandomFeatures = () => {
+  const shuffleFeaturesArray = shuffle(FEATURES);
+  const rundomNumberFeatures = getNumberFeatures();
+  const rundomArrayFeatures = shuffleFeaturesArray.slice(rundomNumberFeatures);
+  return rundomArrayFeatures;
+};
+
+// get random photo hotels
+
+const getRandomPhotos = () => {
+  const randomNumberPhotos = getRandomNumber(0, PHOTOLINKS.length);
+  const photoHotels = [];
+
+  for (let i = 0; i < randomNumberPhotos; i++) {
+    photoHotels.push(`https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/${PHOTOLINKS[i]}`);
   }
 
-  return Number((Math.random() * (max - min) + min).toFixed(floatPoint));
-}
+  return photoHotels;
+};
+
+// out add
+
+const outAdd = () => {
+  const location = {
+    'lat': getFloatingPointNumber(minLat, maxLat, 5),
+    'lng': getFloatingPointNumber(minLng, maxLng, 5),
+  };
+
+  const author = {
+    'avatar': `img/avatars/user${getNumberAvatar()}.png`,
+  };
+
+  const offer = {
+    'title': TITLES[getRandomNumber(0, TITLES.length - 1)],
+    'adress': `${location['lat']}.lat, ${location['lng']}.lng`,
```

Comment:
```suggestion
    'adress': `${location.lat}.lat, ${location['lng']}.lng`,
```
отдавай предпочтение точковому синтаксису)